cmd/admin-bucket-quota.go

```diff
@@ -143,16 +133,11 @@ func mainAdminBucketQuota(ctx *cli.Context) error {
 	// Create a new MinIO Admin Client
 	client, err := newAdminClient(aliasedURL)
 	fatalIf(err, "Unable to initialize admin connection.")
-	quotaStr := ctx.String("fifo")
-	if ctx.IsSet("hard") {
-		quotaStr = ctx.String("hard")
-	}
+
 	_, targetURL := url2Alias(args[0])
-	if ctx.IsSet("fifo") || ctx.IsSet("hard") && len(args) == 1 {
-		qType := madmin.FIFOQuota
-		if ctx.IsSet("hard") {
-			qType = madmin.HardQuota
-		}
+	if ctx.IsSet("hard") && len(args) == 1 {
```

Comment:
I think len(args) == 1 is not needed as well as checkAdminBucketQuotaSyntax() ensures it already